### PR TITLE
Update Sponsor buttons

### PR DESCRIPTION
**Pull request by saundefined**
@pronskiy what do you think about the buttons under the heading -
is it worth hiding the "donate" button that duplicates the CTA in the header on desktop devices, or is it better to leave both always?

I'll also leave it up to you to name these buttons :)

### CODE REVIEW

**Comment by pronskiy** [comment]
Folks, I've added Contact and Donate pages because they make sense regardless of the buttons, and were on my todo already. Then I changed the Donate button to link to the corresponding page.

As for the "Write us", I don't think we need that button, because the contact page feels a more expected way to find out how to reach out, and we'll have it on the top.

File: source/_layouts/default.html [hunk 1]
```diff
@@ -89,15 +89,16 @@
...
         <div class="hidden lg:flex lg:flex-1 lg:justify-end items-center gap-8">
             <a href="https://php.net" target="_blank" rel="noopener noreferrer"
                class="text-sm font-semibold leading-6 text-gray-700 nav-org hover:border-none hover:underline">
                 php.net</a>
 
-            <a href="mailto:contact@thephp.foundation"
-               class="inline-block py-2 no-underline px-6 !text-white bg-[#7f52ff] rounded-3xl hover:bg-[rgba(127,82,255,.8)]">
-                Write Us</a>
+            <a href="/donate" target="_blank" rel="noopener noreferrer"
+               class="inline-block py-2 no-underline px-6 !text-white bg-[#6858f6] rounded-3xl hover:bg-[rgba(104,88,246,.8)]">Donate</a>
```
Comment (saundefined) [inline source/_layouts/default.html:101]:
```suggestion
            <a href="/donate" class="inline-block py-2 no-underline px-6 !text-white bg-[#6858f6] rounded-3xl hover:bg-[rgba(104,88,246,.8)]">Donate</a>
```

File: source/index.html [hunk 2]
```diff
@@ -16,6 +16,11 @@
                 <h1 class="max-w-2xl text-5xl font-bold tracking-tight text-gray-900 sm:text-6xl">
                     We support, advance, and develop the PHP Language
                 </h1>
+                <div class="mt-16 flex flex-row gap-4">
+                    <a href="/donate" target="_blank" rel="noopener noreferrer" class="inline-block lg:hidden text-xl py-2 no-underline px-6 !text-white bg-[#6858f6] rounded-3xl hover:bg-[rgba(104,88,246,.8)]">
```
Comment (saundefined) [inline source/index.html:20]:
```suggestion
                    <a href="/donate" class="inline-block lg:hidden text-xl py-2 no-underline px-6 !text-white bg-[#6858f6] rounded-3xl hover:bg-[rgba(104,88,246,.8)]">
```

**Review by saundefined — COMMENTED** [review]
Looks good! 👍

**Pull request merged**